README.md

```diff
@@ -16,8 +16,32 @@ This repository is provided as an installable python package
     ```
 
 ## Usage
```

Comment:
Add something here about needing to log in to AWS via SSO for `start()` to work.